pnpm-workspace.yaml

```diff
@@ -9,3 +9,43 @@ packages:
     - 'js/private/test/*'
     - 'npm/private/test'
     - 'npm/private/test/*'
+
+onlyBuiltDependencies:
+    - '@aspect-test/c'
+    - '@figma/nodegit'
+    - '@kubernetes/client-node'
+    - 'bufferutil'
+    - 'esbuild'
+    - 'fsevents'
+    - 'segfault-handler'
+    - 'puppeteer'
+    - 'protoc-gen-grpc-ts'
+
+packageExtensions:
+    mocha:
+        peerDependencies:
+            'mocha-multi-reporters': '*'
+    mocha-multi-reporters:
+        peerDependencies:
+            'mocha-junit-reporter': '*'
+    segfault-handler:
+        dependencies:
+            'node-gyp': '*'
+    unix-dgram:
+        dependencies:
+            'node-gyp': '*'
+    '@kubernetes/client-node':
+        # these deps are from devDependencies of @kubernetes/client-node and needed for 'prepare' lifecycle hook
```

Comment:
omg, comments instead of `"// 1": ` json properties 😲

Reply:
that's some fancy 21st century tech